include/re_sip.h

```diff
@@ -297,7 +299,8 @@ int  sip_listen(struct sip_lsnr **lsnrp, struct sip *sip, bool req,
 		sip_msg_h *msgh, void *arg);
 int  sip_debug(struct re_printf *pf, const struct sip *sip);
 int  sip_send(struct sip *sip, void *sock, enum sip_transp tp,
-	      const struct sa *dst, struct mbuf *mb);
+	      const struct sa *dst, struct mbuf *mb,
+	      sip_conn_h *connh, void *arg);
```

Comment:
This breaks the public API.

perhaps add a new function called e.g. `sip_send_conn` ?

Reply:
Okay, will try this.